lib/ohai/common/dmi.rb

```diff
@@ -122,7 +122,7 @@ def convenience_keys(dmi)
 
           records[:all_records].each do |record|
             record.each do |field, value|
-              next if value.is_a?(Mash)
+              next unless value.is_a?(String)
```

Comment:
The Windows API returns values other than strings, which seemed to break an expectation of the values coming from the existing dmi plugin. There is a call to `value.strip` a few lines down so it seemed like a test for String was more in line with the intent here.